tests/test_api.py

```diff
@@ -1525,7 +1524,7 @@ def test_from_id_is_individual(self):
         trx_date = datetime_utcnow()  # After this datetime no transactions should be created
 
         # Check 'from_id' parameter
```

Comment:
I think the name and the description of this test should be updated to `test_from_uuid_is_individual`. This line comment should be updated too with the new parameter name.